cli/src/commit_templater.rs

```diff
@@ -1394,8 +1394,15 @@ fn builtin_tree_diff_methods<'repo>() -> CommitTemplateBuildMethodFnMap<'repo, T
             let path_converter = language.path_converter;
             let template = self_property
                 .map(move |diff| {
+                    let to_tree = diff.to_tree.clone();
```

Comment:
nit: can you add a TODO comment?

The diff object knows `to_tree`, so it can be somehow obtained from `self` without cloning. Since `show_diff_summary()` needs both `tree_diff` and `to_tree`, it might be better to pass `from_tree, to_tree` pair in place of `tree_diff`.

Reply:
I was thinking the same thing.  I added a TODO, but I will send the follow up for this next